pwiz_tools/Skyline/Skyline.cs

```diff
@@ -2451,37 +2451,46 @@ public bool ChangeSettingsMonitored(Control parent, string message, Func<SrmSett
                     bool success = false;
                     using (var longWaitDlg = new LongWaitDlg(this)
                     {
-                        Text = Text,    // Same as dialog box
+                        Text = Text, // Same as dialog box

```

Comment:
Can you revert these ReSharper autoformatting changes to make the important change clearer?